File: app/src/main/java/com/radio/app/models/PlayProgress.java
package com.radio.app.models;

public class PlayProgress {
    private String episodeId;
    private long progress;
    private long recordedAt;

    public PlayProgress() {}
    public PlayProgress(String episodeId, long progress, long recordedAt) {
        this.episodeId = episodeId; this.progress = progress; this.recordedAt = recordedAt;
    }
    public String getEpisodeId() { return episodeId; } public void setEpisodeId(String episodeId) { this.episodeId = episodeId; }
    public long getProgress() { return progress; } public void setProgress(long progress) { this.progress = progress; }
    public long getRecordedAt() { return recordedAt; } public void setRecordedAt(long recordedAt) { this.recordedAt = recordedAt; }
}

File: app/src/main/java/com/radio/app/models/PlayProgress.kt
package com.radio.app.models

data class PlayProgress(
    var episodeId: String? = null,
    var progress: Long = 0L,
    var recordedAt: Long = 0L
) {
    constructor(episodeId: String, progress: Long, recordedAt: Long) : this() {
        this.episodeId = episodeId
        this.progress = progress
        this.recordedAt = recordedAt
    }
}
